tululbot/commands.py

```diff
@@ -43,6 +44,12 @@ def who(message):
     return bot.reply_to(message, about_text, disable_preview=True)
 
 
+@bot.message_handler(commands=['hotline'])
+def hotline(message):
+    app.logger.debug('Detected as hotline command')
+    return bot.forward_message(message.chat.id, message.chat.id, HOTLINE_MESSAGE_ID)
```

Comment:
What happens if `HOTLINE_MESSAGE_ID` is an empty string (the default if not set)? Will this line raise error?